### PR TITLE
Elbow point

### PR DESCRIPTION
**Pull request by airfield20**
Please check cluster.cpp file for mistakes/optimizations.
I also added travis-ci tools, just to try out. Let me know what you guys think.

---
This change is [Reviewable](https://reviewable.io/reviews/iscumd/obstacle_detection/9)

### CODE REVIEW

**Comment by airfield20** [comment]
As of right now the Build passes even with failed test cases.

File: lidar_obstacle_detection/tests/clusterTest.cpp [hunk 1]
```diff
@@ -2,37 +2,87 @@
 // Created by aaron on 06/05/18.
 //
 
+#define MAX_RAND_NUM 25
+
 #include "cluster.h"
 
 #include "catch.hpp"
```
Comment (davidcutting) [inline lidar_obstacle_detection/tests/clusterTest.cpp:9]: "catch.cpp"?

**Review by davidcutting — APPROVED** [review]
Looks good to me. I don't really understand any of the CI stuff, but the layout is good. My only suggestion would be to add small comments in the clustering part as an explanation of what you're doing.

File: lidar_obstacle_detection/tests/clusterTest.cpp (hunk 1 above)
Comment (airfield20) [reply lidar_obstacle_detection/tests/clusterTest.cpp:9]: Its supposed to be hpp

**Pull request merged**